datashape/tests/test_parser.py

```diff
@@ -5,13 +5,21 @@
 from __future__ import absolute_import, division, print_function
 
 import unittest
+import pytest
 
 import datashape
 from datashape.parser import parse
 from datashape import coretypes as ct
 from datashape import DataShapeSyntaxError
 
+
+@pytest.fixture
+def sym():
+    return datashape.TypeSymbolTable()
+
+
 class TestDataShapeParseBasicDType(unittest.TestCase):
```

Comment:
is there a reason we are mixing `py.test` with `unittest` style test cases in this file?